packages/react-navigation/src/__tests__/rum/instrumentation/DdRumReactNavigationTracking.test.tsx

```diff
@@ -495,6 +495,58 @@ describe.each([
                     expect(DdRum.startView).toHaveBeenCalledTimes(2);
                 });
 
+                it('restarts last view when app goes into foreground after being inactive', async () => {
+                    // GIVEN
+                    const navigationRef = createRef<any>();
+                    render(<FakeNavigator1 navigationRef={navigationRef} />);
+
+                    DdRumReactNavigationTracking.startTrackingViews(
+                        navigationRef.current
+                    );
+
+                    // WHEN
+                    appStateMock.changeValue('background');
```

Comment:
why do we need an explicit transition to `background` in this test?

Reply:
I will rename the test. By "goes  into foreground" I meant going from `background` to `active`.